modules/manage/pages/mountable-topics.adoc

```diff
@@ -4,4 +4,6 @@
 :page-categories: Management
 :env-linux: true
 
+For topics with Tiered Storage enabled, you can unmount a topic to safely detach it from a cluster and keep the topic data in the cluster's object storage bucket or container. You can mount the detached topic to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up system resources taken up by the topic, or migrate a topic to a different cluster.
```

Comment:
@towfiqa This is the overview for the self-managed version of the doc. The cloud version will have this, could you review and let me know if any changes are needed?:

(preview available [here](https://deploy-preview-874--redpanda-docs-preview.netlify.app/redpanda-cloud/manage/mountable-topics/?tab=tabs-5-cloud-api))

> For topics with Tiered Storage enabled, you can unmount a topic to safely detach it from a cluster and keep the topic data in the cluster’s object storage bucket or container. You can remount the detached topic to the origin cluster, allowing you to hibernate a topic and free up system resources taken up by the topic.